src/L2DebtManager.sol

```diff
@@ -261,8 +266,33 @@ contract L2DebtManager is
     /**
      * @inheritdoc IL2DebtManager
      */
-    function totalBorrowingAmount() external view returns (uint256) {
-        return _totalBorrowingAmount;
+    function totalBorrowingAmount() public view returns (uint256) {
```

Comment:
Does it keep increasing over time with the fixed borrowing amount due to the interest rate?
How is Aave doing?

Reply:
Then, does it compound?

Reply:
It seems this is how it is done in Credit card industry; https://www.experian.com/blogs/ask-experian/is-credit-card-interest-compounded-daily/

Reply:
> Does it keep increasing over time with the fixed borrowing amount due to the interest rate? How is Aave doing?
Yes it does keep increasing with time. We have a borrow APY per second which is used to calculate the interest.

Reply:
> It seems this is how it is done in Credit card industry; https://www.experian.com/blogs/ask-experian/is-credit-card-interest-compounded-daily/

I think this is what we are also doing. We don't add the balances daily to the user account but the multiplier keeps on growing whenever any user borrows. So its practically compounding every time any user borrows.